driver/registry_base.go

```diff
@@ -370,7 +379,7 @@ func (m *RegistryBase) newKeyStrategy(key string) (s jwk.JWTStrategy) {
 }
 
 func (m *RegistryBase) AccessTokenJWTStrategy() jwk.JWTStrategy {
-	if m.atjs == nil {
+	if m.atjs == nil && m.C.IsUsingJWTAsAccessTokens() {
```

Comment:
This could panic if we request the method but forget to set `IsUsingJWTAsAccessTokens` to true. Do we need this check?

Reply:
Good that you noticed that change I should have highlighted that change! The reasoning behind this was that if jwt access token strategy is not used, then hydra.jwt.access-token key should not be generated. Hardware Security Module might have policy, that restricts generating new keys meaning keys are generated by HSM admin beforehand.  So if opaque access key stradegy is used we are forcing to generate (or pre generate by HSM admin) hydra.jwt.access-token even if its not used. Maybe cloud HSM provider will charge you extra for this also? So I removed this check for now but it is point to consider.